actionview/lib/action_view/test_case.rb

```diff
@@ -263,9 +263,15 @@ def _routes
       end
 
       def method_missing(selector, *args)
-        if @controller.respond_to?(:_routes) &&
-          ( @controller._routes.named_routes.route_defined?(selector) ||
-            @controller._routes.mounted_helpers.method_defined?(selector) )
+        begin
+          routes = @controller.respond_to?(:_routes) && @controller._routes
+          call_routes = routes &&
+            ( routes.named_routes.route_defined?(selector) ||
+            routes.mounted_helpers.method_defined?(selector) )
+        rescue
```

Comment:
I remember someone removing exception swallowing from `after_commit`, I can't quite remember who it was, but I think he might not like this :trollface: